src/client/java/teammates/client/scripts/DataMigrationForOrphanedStudentProfile.java

```diff
@@ -1,52 +0,0 @@
-package teammates.client.scripts;
```

Comment:
Any reason  for deleting this file?

Reply:
That's the fair for our live server, but if let's say if somebody runs their private teammate server then we cannot know when they will run the script. Or developers running a staging server and want to upgrade their version without losing the created toy data etc. So the best is we should keep the scripts and document them.

Reply:
That is a fair point. In fact we used to do that for a long time and ended up with a whole bunch of scripts that we needed to maintain e.g., when a datastore API or java version is updated. At some point we got fed up and decided to delete scripts as soon as we are done with them. We don't have any others running production servers ATM AFAIK. If a dev needs a script to upgrade a staging server (also very rare), I guess they can use git to retrieve the script from history.

Reply:
I see. We don't have the manpower to maintain migration scripts and nobody is complaining that their database breaks, so we need to be practical and spend resources on more important things.

Reply:
Yup, that's the situation.